packages/replay/src/rrweb-types.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
+type blockClass = string | RegExp;
```

Comment:
I would maybe put this into `src/types/rrweb.ts`, what do you think? Then we can also put some other stuff in there, keep the types together?

Reply:
Sounds good - [5e57422](https://github.com/getsentry/sentry-javascript/pull/6464/commits/5e57422b51ff0eeb3c72cc7678d04282e782a3db)